### PR TITLE
chore: debug log plugin phases

### PR DESCRIPTION
**Pull request by rwoll**

### CODE REVIEW

File: packages/playwright-test/src/util.ts [hunk 1]
```diff
@@ -294,3 +294,5 @@ export async function normalizeAndSaveAttachment(outputPath: string, name: strin
     return { name, contentType, body: typeof options.body === 'string' ? Buffer.from(options.body) : options.body };
   }
 }
+
+export const pluginLogger = (plugin: TestPlugin) => debug(`pw:plugin:${plugin.name}`);
```
Comment (mxschmitt) [inline packages/playwright-test/src/util.ts:298]: lets use pw-test instead of pw? Since its more test-runner specific.

Comment (rwoll) [inline packages/playwright-test/src/util.ts:298]: This is keeping it simple. I almost wrapped our plugins with a Proxy so we would log accesses to the interesting methods, but that became silly.

Comment (rwoll) [reply packages/playwright-test/src/util.ts:298]: Is there a precedent for this? I frequently just do `pw:*` expecting to get all the relevant logs

Comment (rwoll) [reply packages/playwright-test/src/util.ts:298]: There is a `export const debugTest = debug('pw:test');` in a few places, so I will scope it from there

**Comment by rwoll** [comment]
Closing. We'll revive when we want to land user-facing plugin API.

**Comment by rwoll** [comment]
Closing. We'll revive when we want to land user-facing plugin API.

**Pull request closed without merging**